Frends.Kafka.Produce/Frends.Kafka.Produce/Definitions/Ssl.cs

```diff
@@ -118,34 +118,34 @@ public class Ssl
     /// <example>ExamplePassword</example>
     [UIHint(nameof(UseSsl), "", true)]
     [PasswordPropertyText]
-    public string SslKeystorePassword { get; set; } = "";
+    public string SslKeystorePassword { get; set; }
 
     /// <summary>
     /// Private key passphrase for use with Ssl.SslKeyLocation)
     /// </summary>
     /// <example>ExamplePassword</example>
     [UIHint(nameof(UseSsl), "", true)]
     [PasswordPropertyText]
-    public string SslKeyPassword { get; set; } = "";
+    public string SslKeyPassword { get; set; }
 
     /// <summary>
     /// A cipher suite is a named combination of authentication, encryption, MAC and key exchange algorithm used to negotiate the security settings for a network connection using TLS or SSL network protocol.
     /// </summary>
     /// <example>foo</example>
     [UIHint(nameof(UseSsl), "", true)]
-    public string SslCipherSuites { get; set; } = "";
+    public string SslCipherSuites { get; set; }
 
     /// <summary>
     /// The supported-curves extension in the TLS ClientHello message specifies the curves (standard/named, or 'explicit' GF(2^k) or GF(p)) the client is willing to have the server use.
     /// </summary>
     /// <example>1</example>
```

Comment:
Is this a good example? I have no idea what this field does, but look like "1" is not a valid value for this.